libavformat/dashdec.c

```diff
@@ -422,7 +422,7 @@ static void free_subtitle_list(DASHContext *c)
 
 static int open_url(AVFormatContext *s, AVIOContext **pb, const char *url,
                     AVDictionary **opts, AVDictionary *opts2, int *is_http,
-                    const AVStream* stream, uint64_t* filesize)
+                    const AVStream* stream)
```

Comment:
Was the addition of `filesize` as an argument something we added in the past? I don't remember.

Reply:
Yes, it was added by my previous change.  No longer necessary.